Sources/FeaturevisorSDK/Instance+Feature.swift

```diff
@@ -35,6 +35,49 @@ extension FeaturevisorInstance {
             return true;
         });
     }
+
+    func getMatchedAllocation(
+            traffic: Traffic,
+            bucketValue: Int) -> Allocation? {
```

Comment:
out of curiosity, is SwiftFormat automatically putting the closing `)` in the same line?

Reply:
Actually, I would need to check the defult one. For years I stuck to https://github.com/nicklockwood/SwiftFormat and the current rules that we have in my current company